src/main/scala/com/cognite/sdk/scala/v1/files.scala

```diff
@@ -91,6 +92,7 @@ final case class FileCreate(
 
 final case class FileUpdate(
     externalId: Option[Setter[String]] = None,
+    instanceId: Option[Setter[InstanceId]] = None,
```

Comment:
this one isn't supported I think, or at least
- looking at https://api-docs.cognite.com/20230101/tag/Files/operation/updateFiles instanceId can't be changed
- WithSetExternalId trait corresponds to externalId one line above, but we don't have similar thing for instanceId
- files client has updateById&updateByExternalId but not update by instanceId, but we probably need it?

Reply:
You are right, this is an oversight.

We will need update eventually, it's not included in this PR.